newton/examples/mpm/example_mpm_anymal.py

```diff
@@ -112,10 +112,13 @@ def __init__(
         for key, value in initial_q.items():
             builder.joint_q[builder.joint_key.index(key) + 6] = value
 
-        for i in range(builder.joint_dof_count):
-            builder.joint_dof_mode[i] = newton.JointMode.TARGET_POSITION
-            builder.joint_target_ke[i] = 150
-            builder.joint_target_kd[i] = 5
+        for i in range(len(builder.joint_target_ke)):
+            if i < 6:
```

Comment:
Can you add a comment here that we skip the free joint dofs, just to make these examples easier to follow. I think also this `+ 6` line before would be good to clarify.